Open Judge System/Workers/OJS.Workers.ExecutionStrategies/SqlStrategies/MySql/MySqlRunSkeletonRunQueriesAndCheckDatabaseExecutionStrategy.cs

```diff
@@ -10,7 +10,7 @@ public class MySqlRunSkeletonRunQueriesAndCheckDatabaseExecutionStrategy : BaseM
         {
         }
 
-        public ExecutionResult Execute(ExecutionContext executionContext)
+        public override ExecutionResult Execute(ExecutionContext executionContext)
```

Comment:
The execution method in the inheriting strategies differs from child to child, so it cannot be made virtual in the base class.